shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -495,13 +496,55 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
     lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_CREATE);
```

Comment:
Should existing use of `lifecycle` change to `getLifecycle()`? Otherwise it seems like we're introducing the possibility of NPEs.